packages/babel-parser/test/fixtures/flow/class-properties/declare-static-field/input.js

```diff
@@ -0,0 +1,3 @@
+class A {
+  declare static foo
+}
```

Comment:
Can you add a new test case swapping `declare` and `static`? It is allowed in TypeScript but not flow.
```js
class A {
  static declare foo
}
```